packages/launchpad/src/checklist-item/style.scss

```diff
@@ -259,3 +280,13 @@
 		}
 	}
 }
+
+.checklist-item__expanded-content {
+	padding-left: 2px;
```

Comment:
This just matches the `padding-left` from the button. I didn't want to move this to the `li` just to avoid breaking something, so I added the same spacing here.